src/Qowaiv.Data.SqlClient/Sql/Timestamp.cs

```diff
@@ -6,6 +6,9 @@
 [OpenApiDataType(description: "SQL Server timestamp notation.", example: "0x00000000000007D9", type: "string", format: "timestamp")]
 [OpenApi.OpenApiDataType(description: "SQL Server timestamp notation.", example: "0x00000000000007D9", type: "string", format: "timestamp")]
 [TypeConverter(typeof(Conversion.Sql.TimestampTypeConverter))]
+#if NET5_0_OR_GREATER
```

Comment:
I see both "NET5_0_OR_GREATER" and "NET6_0_OR_GREATER" - is this intended?

Reply:
No, it is not.